Sources/SwiftParser/README.md

```diff
@@ -0,0 +1,70 @@
+## Overview
```

Comment:
(not a blocker) The existence of this file triggers a warning, at least with 5.7:

```
warning: 'swift-syntax': found 1 file(s) which are unhandled; explicitly declare them as resources or exclude from the target
    Sources/SwiftParser/README.md
```

Reply:
I'll merge and we'll fix it up afterward. Thanks!